<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_page</class>
 <widget class="QWidget" name="settings_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>369</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#FFF2D8</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="MainSettingsContents">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="SettingsTabWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="SettingsTab1">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
            </property>
            <property name="text">
             <string>General Settings</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="GeneralSettignsTitleBar_2">
            <property name="styleSheet">
             <string notr="true">QGroupBox{border:0px;}
QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}

</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>50</number>
                </property>
                <property name="rightMargin">
                 <number>50</number>
                </property>
                <item>
                 <widget class="QLabel" name="displayNameLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>95</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
                  </property>
                  <property name="text">
                   <string>Change Display Name</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="DisplayNameLineEdit"/>
                </item>
                <item>
                 <widget class="QPushButton" name="diaplayNameChangeButton">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>246</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
                  </property>
                  <property name="text">
                   <string>Change Theme</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>115</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QCheckBox" name="loginCheckBox">
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
                  </property>
                  <property name="text">
                   <string>Start up at login    :</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="trayCheckBox">
                  <property name="layoutDirection">
                   <enum>Qt::RightToLeft</enum>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
                  </property>
                  <property name="text">
                   <string>Minimize to tray on close    :</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="SettingsTab2">
         <attribute name="title">
          <string>Semester Settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
            </property>
            <property name="text">
             <string>Semester Settings</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="GeneralSettignsTitleBar">
            <property name="styleSheet">
             <string notr="true">QGroupBox{border:0px;}
</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="styleSheet">
                <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Current year</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="current_year">
                  <property name="text">
                   <string>#</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="currentSemesterlol">
                  <property name="text">
                   <string>Current Semester</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="currentSemester">
                  <property name="text">
                   <string>#</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Semester Per year</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="semesterPerYear">
                  <property name="text">
                   <string>#</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QPushButton" name="ascendSemesterButton">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}

</string>
                  </property>
                  <property name="text">
                   <string>Ascend Semester</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rollBackSemester">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}

</string>
                  </property>
                  <property name="text">
                   <string>Rollback Semester</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="manualModifyButton">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}

</string>
                  </property>
                  <property name="text">
                   <string>Manual Modify</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_7">
               <property name="styleSheet">
                <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
               </property>
               <property name="title">
                <string>Modify a Course below</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="topMargin">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QListWidget" name="course_list">
                  <property name="styleSheet">
                   <string notr="true">border:1px solid black;
border-radius:10px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_8">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
  background-color: #fff;
  border: 0 solid #e2e8f0;
  border-radius: 5px;
  color: #0d172a;
  font-family: &quot;Basier circle&quot;,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 5px 5px;
  text-align: center;
  text-decoration:none #0d172a solid;
}

QPushButton:hover {
  background-color: #221D23;
  color: #fff;
}

</string>
                  </property>
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QPushButton" name="courseSettingRename">
                     <property name="text">
                      <string>Rename</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="courseSettingDelete">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="courseSettingExtra">
                     <property name="text">
                      <string>PushButton</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="FileSettings">
         <attribute name="title">
          <string>File Settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px solid black;
border-radius: 10px;
font-size: 15px;
  font-weight: 600;
  line-height: 1;
  padding: 2px 2px;
color:black;
</string>
            </property>
            <property name="text">
             <string>File Settings</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="GeneralSettignsTitleBar_3">
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
